packages/kas/README.md

```diff
@@ -0,0 +1,94 @@
+KAS
+===
+
+A lightweight JavaScript CAS for comparing expressions and equations.
```

Comment:
For a future PR, it'd be nice to expand what `CAS` is... I have no idea.

Reply:
CAS = Computer Algebra System